packages/scripts/src/utils/contracts.ts

```diff
@@ -13,10 +13,8 @@ export const getContractInstance = async <T>(
   chainId?: number
 ): Promise<T> => {
   const networkId = chainId ? chainId : await hre.getChainId();
-  const factory = await hre.ethers.getContractFactory(contractName);
```

Comment:
Factory is not available for abstract contracts, e.g. when using `Multicall2`. Since this method is not used for deployments we can use `hre.ethers.getContractAt` instead